praw/models/reddit/subreddit.py

```diff
@@ -873,6 +877,120 @@ def submit(
 
         return self._reddit.post(API_PATH["submit"], data=data)
 
+    def submit_gallery(
+        self,
+        title,
+        images,
+        flair_id=None,
+        flair_text=None,
+        resubmit=True,
+        send_replies=True,
+        nsfw=False,
+        spoiler=False,
+        collection_id=None,
+        discussion_type=None,
```

Comment:
Can you sort all the params with defaults please?

Reply:
This was copied from `submit_image` do you want those to be sorted as well?

Reply:
Changing the order will require a breaking change in the other places. Though maybe we should make these mandatory keyword arguments on this function so keeping things sorted in the future doesn't break anything. It looks like that feature was supported since 3.0. Can you try that here and see if it works?

    def compare(a, b, *, key=None):